platform/android/scripts/generate-style-code.mjs

```diff
@@ -0,0 +1,470 @@
+import fs from "node:fs";
+import ejs from "ejs";
+import spec from "./android-style-spec-overrides.mjs";
+import _ from "lodash";
+
+// FIXME: https://github.com/mapbox/mapbox-gl-native/issues/15008
+//delete spec.layout_circle["circle-sort-key"]
+//delete spec.layout_line["line-sort-key"]
+//delete spec.layout_fill["fill-sort-key"]
+
+import { writeIfModified, camelize, snakeCaseUpper, iff, camelizeWithLeadingLowercase } from "../../../scripts/style-code.mjs";
+
+function setupGlobalEjsHelpers() {
```

Comment:
Not sure I understand why this is needed... Why adding methods to the global object?

Reply:
EJS needs it.